backend/rest/rest.go

```diff
@@ -35,53 +38,56 @@ func New(token string, resultsPerPage int, storageProvider persistence.Storage)
 	return &client{githubClient, ctx, resultsPerPage}, nil
 }
 
-// Search searches code using the github api
-func (c *client) Search(query string, lang string) map[int64]*backends.Result {
+var repos map[int64]*backends.Result
```

Comment:
This is a file-scoped variable, yet it's between 2 functions. Is that standard way in GO?

Reply:
done